docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -78,7 +78,7 @@ Docker: Start a Multiple-node Sawtooth Network
 
 In this procedure, you will use a Docker Compose file that creates a new
 application development environment with five validator nodes and four
-transaction processors (Settings, IntegerKey, XO, and PoET Validator Registry).
+transaction processors (Settings, IntegerKey, XO, and Validator Registry).
```

Comment:
I do not think this is a good change to make, as it will be unclear to those with a strong knowledge of Sawtooth that this "Validator Registry" is a PoET specific transaction processor. If consistency is needed, then we should make the change in the spec.

Reply:
I could restore "PoET" here, but then it will be inconsistent with the rest of the Sawtooth docs. The transaction family spec calls it "Validator Registry", as does the rest of the doc set (mostly).  FYI: This was the original name of the V.R. txn family spec in the doc - this chapter never had "PoET" in the title.  For examples, see: 
https://sawtooth.hyperledger.org/docs/core/nightly/master/transaction_family_specifications.html
https://sawtooth.hyperledger.org/docs/core/nightly/master/transaction_family_specifications/validator_registry_transaction_family.html

If "PoET" is needed in the name here, it is also needed everywhere else. If so, I suggest a separate PR to make the term consistent everywhere. Note that this affects the images in the app dev guide as well as the Txn Fam spec.  I think some CLI "code" (help output) and config files will also need to change.  I suspect that the doc in the separate PoET repo might need changing, too.